fairlearn/postprocessing/_interpolated_thresholder.py

```diff
@@ -21,14 +21,13 @@ class InterpolatedThresholder(BaseEstimator, MetaEstimatorMixin):
 
     At prediction time, the predictor takes as input both standard and sensitive features.
     Based on the values of sensitive features, it then applies a randomized thresholding
-    transformation according to the provided `interpolation_dict`.
+    transformation according to the provided `threshold_interpolation`.
```

Comment:
why this change? if you don't like the original, we should go for something more concise. maybe something like `threshold_info` or `thresholding_info` or even `thresholds` (but that is a bit confusing, because it sounds like the structure would be simpler).

Reply:
Info isn't a great name. Everything is "info", yet we don't call "width" variables "width_info". It's a stylistic improvement as well as a description improvement: interpolation and threshold are the two important things worth mentioning IMO.
If you don't think it's an improvement I don't mind scrapping it, though (not the hill I choose to die on). It was originally part of an older PR about documenting post processing (which I'm actively working on) and got refactored out into its own change for simplicity.

Reply:
How about `threshold_weights` since basically each value of a threshold gets a certain probabilistic weight?

Reply:
I don't dislike it, but that's just p0 and p1 🙂 

It's really not a bad name though...